cart/server.js

```diff
@@ -117,44 +130,50 @@ app.delete('/cart/:id', (req, res) => {
             throw new Error('Invalid cart id: ' + req.params.id);
         }
     } catch(e) {
-        logger.error(e);
-        res.status(500).send(e.message);
+        req.log.error(e);
```

Comment:
Let's retain the `logger.error(e)` to navigate to the cart error logs (in Loki) from Workbench

Reply:
That looks like a typo on my part. Should have been `req.log` and not `logger`.

Reply:
This is inside the `delete` handler.

`req.log` is the same as `logger` both write the log to the same place.
`req.log` is an express wrapped version of logger. See from line 24.